src/test/java/org/fundacionjala/coding/melvi/MovieRefactorTest.java

```diff
@@ -0,0 +1,26 @@
+package org.fundacionjala.coding.melvi;
+
+import org.fundacionjala.coding.melvi.MovieRefactor.Customer;
+import org.fundacionjala.coding.melvi.MovieRefactor.Rental;
+import org.fundacionjala.coding.melvi.MovieRefactor.Movie;
+import org.fundacionjala.coding.melvi.MovieRefactor.PriceCalculationRegularClient;
+import org.fundacionjala.coding.melvi.MovieRefactor.ReportGenFullFormat;
+import org.fundacionjala.coding.melvi.MovieRefactor.PriceCodeEnum;
+import org.junit.Test;
+
+/**
+ *
+ */
+public class MovieRefactorTest {
+
+    /**
+     *
+     */
+    @Test
+    public void testRentals() {
+        Customer customer = new Customer("Test", new PriceCalculationRegularClient(), new ReportGenFullFormat());
+        customer.addRental(new Rental(new Movie("The Revenant", PriceCodeEnum.NEW_RELEASE), 2));
+        customer.addRental(new Rental(new Movie("Terminator", PriceCodeEnum.REGULAR), 2));
+        System.out.println(customer.statement());
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Replace this use of System.out or System.err by a logger. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS106)